docs/source/conf.py

```diff
@@ -75,15 +75,37 @@
 
 """
 
+from sphinx_gallery.sorting import _SortKey
```

Comment:
Private, but if this is ever an issue I think it wil be very easily resolved. Alternatively I can just write the `__init__` in our `CustomGalleryExampleSortKey`

Reply:
Given that this will only be [two lines](https://github.com/sphinx-gallery/sphinx-gallery/blob/74a8f1d5b3dd364fb4f2180b57f4fa9af9db858a/sphinx_gallery/sorting.py#L61-L64), I think we can safely inline it.